legacy/Makefile

```
@@ -10,6 +10,7 @@ clean:
 .PHONY: install
 install:
 	set -e; for i in $(SUBDIRS); do $(MAKE) -C $$i install; done
+	install -m 644 data/install_config $(DESTDIR)/var/lib/restraint
```

This will fail if `$(DESTDIR)/var/lib/restraint` doesn't exist.  If only `$(DESTDIR)/var/lib/` exists, it will copy `install_config` into `$(DESTDIR)/var/lib/restraint`, as a file. At least,
```suggestion
	install -m 644 data/install_config $(DESTDIR)/var/lib/restraint/
```

---

Please refer to the main Makefile, where it explicitly creates the directory.
  install -m0755 -d $(DESTDIR)/var/lib/restraint
A lot will go wrong if this directory does not exist.  The 'restraint' package must be installed prior to 'restraint-rhts'.

---

The main Makefile is not related to this one.